internal/network/command.go

```diff
@@ -453,21 +453,28 @@ func addPhaseFlag(cmd *cobra.Command, resourceType string) {
 	pcmd.RegisterFlagCompletionFunc(cmd, "phase", func(_ *cobra.Command, _ []string) []string {
 		switch resourceType {
 		case resource.NetworkLinkService:
-			return []string{"READY"}
+			return []string{"ready"}
 		case resource.NetworkLinkEndpoint:
-			return []string{"PROVISIONING", "PENDING_ACCEPT", "READY", "FAILED", "DEPROVISIONING", "EXPIRED", "DISCONNECTED", "DISCONNECTING", "INACTIVE"}
+			return []string{"provisioning", "pending_accept", "ready", "failed", "deprovisioning", "expired", "disconnected", "disconnecting", "inactive"}
```

Comment:
Change underscores to dashes and use `utils.ToLower()` and `utils.ToUpper()` to convert between frontend and backend representations
```suggestion
			return []string{"provisioning", "pending-accept", "ready", "failed", "deprovisioning", "expired", "disconnected", "disconnecting", "inactive"}
```